README.md

```diff
@@ -14,55 +14,35 @@ Some of the applications that use this gem:
 
 ### Integration with a Rails 4+ app
 
-To use gds-sso you will need an oAuth client ID and secret for Signon or a compatible system.
-These can be provided by one of the team with admin access to Signon.
+- Include the gem in your Gemfile:
 
-Then include the gem in your Gemfile:
+  ```ruby
+  gem 'gds-sso', '<version>'
+  ```
 
-```ruby
-gem 'gds-sso', '<version>'
-```
-
-Create a `config/initializers/gds-sso.rb` that looks like:
-
-```ruby
-GDS::SSO.config do |config|
-  config.user_model   = 'User'
-
-  # set up ID and Secret in a way which doesn't require it to be checked in to source control...
-  config.oauth_id     = ENV['OAUTH_ID']
-  config.oauth_secret = ENV['OAUTH_SECRET']
-
-  # optional config for location of Signon
-  config.oauth_root_url = "http://localhost:3001"
-
-  # Pass in a caching adapter cache bearer token requests.
-  config.cache = Rails.cache
-end
-```
+- Create `config/initializers/gds-sso.rb`:
 
-The user model must include the `GDS::SSO::User` module.
+  ```ruby
+  GDS::SSO.config do |config|
+    # Pass in a caching adapter cache bearer token requests.
+    config.cache = Rails.cache
+  end
+  ```
 
-It should have the following fields:
+- Create a "users" table in the database: ([example migration with all the necessary fields](https://github.com/alphagov/content-publisher/blob/16c58a40745c1ea61ec241e5aeb702ae15238f98/db/migrate/20160622154200_create_users.rb))
```

Comment:
I don't see much of a benefit in linking to this instead of declaring the fields inline. It makes the README shorter, but if we need to remove or rename a field, this commit won't be found in a mass find-and-replace search.

Reply:
I don't follow, sorry. Could you elaborate on the last sentence?

I think there's some advantage in linking to executable code, rather than pseudo-stuff. For one thing, someone can just go and copy paste it directly into their new/updated app.

I do agree that if the fields involved needed to change (very unlikely), then we'd need to link to something else, or inline it again.

Reply:
If the string  "organisation_slug"` got renamed to `organization_slug`, for example, a find-and-replace across the repo would update the example code in the README, but couldn't update the [linked commit contents](https://github.com/alphagov/content-publisher/blob/16c58a40745c1ea61ec241e5aeb702ae15238f98/db/migrate/20160622154200_create_users.rb). If we forgot to update the link to point to something else, we'd be pointing devs to code that is no longer fully compatible.

Agreed it's super unlikely, just thought it was worth questioning why moving content out of the README itself is seen as an improvement.

Reply:
Ah, that's an interesting example. I agree that's a possibility, so a potential disadvantage. Equally, someone (e.g. me) could forget to update the README at all.

I don't particularly mind either way. My original motivation came from the observation that "ugh, this is a long README". Since neither of us seem to feel strongly, I'd prefer to leave it as-is.

Reply:
Sure 👍 I think everything is addressed then! ✅